### ISSUE TITLE
bug: Issue with Visual Block Mode Keybinding When Sourcing mswin.vim

### ISSUE THREAD

**Issue by tj-amano** [vscode-neovim/vscode-neovim#1903]
### Check the following:

- [X] I have checked the behavior after setting "vscode-neovim.neovimClean" in VSCode settings.
- [X] I have read the vscode-neovim docs.
- [X] I have searched existing issues.

### Neovim version (nvim -v)

0.9.5

### Operating system/version

Windows 11 22H2

### Describe the bug

Dear vscode-neovim Developers,

When sourcing `$VIMRUNTIME/mswin.vim` in my `init.vim`, the following mapping (in the mswin.vim) for entering Visual Block mode (rectangular selection) does not seem to work:

```vim
noremap <C-Q> <C-V>
```

### Steps To Reproduce

1. Moving the source $VIMRUNTIME/mswin.vim line to the top of init.vim to ensure the mapping takes priority.
2. Ensuring there are no conflicting keybindings in VSCode for Ctrl+Q.
3. Considering the possibility of interference from VSCode's default Ctrl+V (Paste) keybinding and changing it, but it did not help.

### Expected Behavior

When pressing Ctrl+Q in the VSCode editor area, Neovim should enter Visual Block mode, allowing rectangular selections.
Actual Behavior
The Ctrl+Q mapping does not work, and Neovim does not enter Visual Block mode.
However, when invoking Neovim from the terminal, Ctrl+Q works correctly and enters Visual Block mode.
I suspect that there might be a conflict with the Neovim extension's multi-cursor feature.

Please guide me on the potential cause of this issue and any possible workarounds or solutions.
Let me know if you need any additional information to diagnose the problem.

**Comment by justinmk** [comment]
default vscode-neovim keybinding for ctrl-v:

```
{ "key": "ctrl+v",                "command": "vscode-neovim.send",
                                     "when": "editorTextFocus && neovim.ctrlKeysInsert.v && neovim.init && neovim.mode == 'insert' && editorLangId not in 'neovim.editorLangIdExclusions'",
                                     "args": "<C-v>" },
```

ctrl-q:

```
{ "key": "ctrl+q",                "command": "vscode-neovim.send",
                                     "when": "editorTextFocus && neovim.ctrlKeysNormal.q && neovim.init && neovim.mode != 'insert' && editorLangId not in 'neovim.editorLangIdExclusions'",
                                     "args": "<C-q>" },
```

So, doesn't seem to be a missing default keybinding.

What happens if you run this from the vscode-neovim `:` cmdline prompt:

    call nvim_input('<c-q>')

**Comment by tj-amano** [comment]
Thank you for the suggestion to test the `call nvim_input('<c-q>')` command from the Neovim command-line prompt within VSCode. I followed your instructions, and the command successfully entered Visual Block mode in the Neovim instance running inside VSCode.

This confirms that the issue lies in how key inputs are being handled and translated between VSCode and Neovim, rather than a problem with Neovim itself or the mapping in mswin.vim.

Since the `<c-q>` input works as expected when sent directly to Neovim, but not when typed in the VSCode editor area, it points to a potential issue in the key input translation logic between the two applications.

Please let me know if you need any additional information or logs to further investigate this issue. I understand that this may require a deeper analysis of the key input handling mechanisms between VSCode and Neovim, and I appreciate your efforts in addressing this problem.

Any workarounds or potential solutions you can provide in the meantime would be greatly appreciated until a more permanent fix can be implemented in a future release of the vscode-neovim extension.

**Comment by justinmk** [comment]
Run the vscode command `Preferences: Open Keyboard Shortcuts` and check if you or an extension has mapped ctrl-q to something else.

**Comment by tj-amano** [comment]
Thank you for the suggestion.

I have already verified that there are no conflicting keybindings assigned to `Ctrl+Q` in my VSCode setup.

Despite this, the issue with the `Ctrl+Q` mapping for entering Visual Block mode in Neovim persists when using the vscode-neovim extension inside the VSCode editor area.

However, as we confirmed, when sending the `<c-q>` input directly to Neovim using the `:call nvim_input('<c-q>')` command, it successfully enters Visual Block mode as expected.

This behavior suggests that the root cause lies in how key inputs are being handled and translated between VSCode and Neovim, rather than a conflicting keybinding issue.
